src/common/Pagination/styled.js

```diff
@@ -0,0 +1,51 @@
+import styled from "styled-components";
+import { ReactComponent as prevArrow } from "./svg's/prevArrow.svg";
+import { ReactComponent as nextArrow } from "./svg's/nextArrow.svg";
+
+export const Wrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 40px;
+  margin-bottom: 100px;
+`;
+
+export const Button = styled.button`
+  display: flex;
+  align-items: center;
+  padding: 8px 12px;
+  margin-right: 12px;
+  background-color: lightblue;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+`;
+
+export const PrevArrowIcon = styled(prevArrow)`
+  margin: 0 4px;
+  color: lightblue;
+`;
+
+export const NextArrowIcon = styled(nextArrow)`
+  margin: 0 4px;
+  color: lightblue;
+`;
+
+export const ButtonText = styled.p`
```

Comment:
I would also make the text "grey"  using the same color as in figma project => rgb(51, 51, 51)
We will later leave it black in disabled buttons